### PR TITLE
docs: add gui docs

### PR DESCRIPTION
**Pull request by devadvocado**

### CODE REVIEW

File: docs/docs/draft/documentation/README.md [hunk 1]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
 * `kumactl help [..]`: help dialog that explains the commands available.
 * `kumactl version [--detailed]`: shows the version of the program.
 
+## GUI
+
+Kuma now ships with a basic web-based GUI that will serve as a visual overview of your data planes, meshes, and various traffic policies.
+
+The GUI pairs with the HTTP API — Read more about the HTTP API here
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's add a link to HTTP API

File: docs/docs/draft/documentation/README.md [hunk 2]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
...
+
+When launching Kuma, the GUI will start by default on port `:5683`. You can access it in your web browser by going to `http://localhost:5683/`.
+
+### Getting Started
+When you run the GUI for the first time, you’ll be presented with a simple walkthrough. This will:
+1. Confirm that Kuma is running in either universal or Kubernetes mode
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's change to `Universal` (capital case)

File: docs/docs/draft/documentation/README.md [hunk 3]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
...
+2. Provide instructions on how to add data planes (if none have yet been added)
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's change `data planes => dataplanes`

File: docs/docs/draft/documentation/README.md [hunk 4]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
 * `kumactl help [..]`: help dialog that explains the commands available.
 * `kumactl version [--detailed]`: shows the version of the program.
 
+## GUI
+
+Kuma now ships with a basic web-based GUI that will serve as a visual overview of your data planes, meshes, and various traffic policies.
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's change `data planes => dataplanes`

File: docs/docs/draft/documentation/README.md [hunk 5]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
...
+
+The GUI pairs with the HTTP API — Read more about the HTTP API here
+
+When launching Kuma, the GUI will start by default on port `:5683`. You can access it in your web browser by going to `http://localhost:5683/`.
+
+### Getting Started
+When you run the GUI for the first time, you’ll be presented with a simple walkthrough. This will:
+1. Confirm that Kuma is running in either universal or Kubernetes mode
+2. Provide instructions on how to add data planes (if none have yet been added)
+3. Provide a short list of data planes found, in order to confirm that things are working accordingly and the app can display information
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's change `data planes => dataplanes`

File: docs/docs/draft/documentation/README.md [hunk 6]
```diff
@@ -358,6 +358,29 @@ Available commands on `kumactl` are:
...
+
+### Global Overview
+Once you’ve completed the setup process, you’ll be sent to the Global Overview. This is a general overview of all of the meshes found. You can then view each entity and see how many data planes and traffic permissions, routes, and logs are associated with that mesh.
```
Comment (yskopets) [inline docs/docs/draft/documentation/README.md]: Let's change `data planes => dataplanes`

File: docs/docs/draft/documentation/README.md (hunk 1 above)
Comment (dhruv) [reply docs/docs/draft/documentation/README.md]: https://kuma.io/docs/latest/documentation/#http-api

**Pull request merged**